src/notification/notification.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@nestjs/common'
+import { PrismaService } from './../database/prisma.service'
+import NotifiInput from './dto/notifi-create.input'
+
+@Injectable()
+export class NotificationService {
+    constructor(private readonly prisma: PrismaService) {}
+
+    async createNotification(createNotificationDto: NotifiInput) {
+        return await this.prisma.notification.create({
+            data: {
+                userId: createNotificationDto.userId,
+                auctionId: createNotificationDto.auctionId,
+                typeNotifi: createNotificationDto.typeNotifi,
+                message: createNotificationDto.message,
+            },
+        })
+    }
+
+    async readNotification(notificationId: number) {
+        return await this.prisma.notification.update({ where: { id: notificationId }, data: { read: true } })
+    }
+
```

Comment:
`findAllUserByAuctionId` `findAllNotificationUserByUserId` use same model, but name of methods is different `User` and `NotificationUser` - you need to choose one. And if you use `findMany` its mean multiply so `NotificationsUser` (add **s**)